Tests/scripts/validate_files_structure.py

```diff
@@ -28,6 +28,7 @@
 import argparse
 from subprocess import Popen, PIPE
 from distutils.version import LooseVersion
+import secrets
```

Comment:
change that to `from secrets import get_secrets` and move it to line 32(PEP8 requires separation between third party imports and regular ones(Will cause more trouble when adding pylint in the future)